docs/guides/terminal.md

```diff
@@ -0,0 +1,78 @@
+[](){#ref-guides-terminal}
+# Terminal usage on Alps
+
+This documentation is a collection of guides, hints and tips for setting up your terminal environment on Alps.
+
+[](){#ref-guides-terminal-shells}
+## Shells
+
+Every user has a shell that will be used when they log in, with [bash](https://www.gnu.org/software/bash/) as the default shell for new users at CSCS.
+
+At CSCS the vast majority of users stick with the default `bash`: at the time of writing, with over 1200 users on Daint, 4 were using C shell, 2 were using zsh, and the rest were using Bash.
+
```

Comment:
```suggestion
```
Is this useful? I'd remove it, it is also bound to go out of date.

Reply:
I will change this to < 1%.
This is likely to not change radically.